### PR TITLE
Bump AGP to 7.3.0 and fix warning

### PR DESCRIPTION
**Pull request by DeKaN**
Part of #185

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -13,6 +13,7 @@ android {
         versionCode 28
         versionName "2.8"
     }
+    namespace = "com.github.muellerma.coffee"
```
Comment (mueller-ma) [inline app/build.gradle:16]: Why is this change needed?

Comment (DeKaN) [reply app/build.gradle:16]:
```
Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated.
Please instead set the namespace (or testNamespace) in the module's build.gradle file, as described here: https://developer.android.com/studio/build/configure-app-module#set-namespace
This migration can be done automatically using the AGP Upgrade Assistant, please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.
```
I fixed this warning during AGP update

**Review by mueller-ma — APPROVED** [review]
